frontend/src/Core/Types.elm

```diff
@@ -3,6 +3,8 @@ module Core.Types exposing (..)
 import Time exposing (Posix)
 import Json.Decode exposing (Value)
 
+type alias ElmStreetNonEmptyList a = (a, List a)
+
```

Comment:
this is now required as we can't produce type annotation `(a, List a)` - but we can produce annotation for this alias.